Userland/Libraries/LibAudio/ConnectionToServer.cpp

```diff
@@ -43,10 +45,13 @@ ConnectionToServer::~ConnectionToServer()
 
 void ConnectionToServer::die()
 {
-    // We're sometimes getting here after the other thread has already exited and its event loop does no longer exist.
-    if (Threading::MutexLocker const locker(m_enqueuer_loop_destruction); m_enqueuer_loop != nullptr) {
-        m_enqueuer_loop->wake();
-        m_enqueuer_loop->quit(0);
+    {
```

Comment:
Same here

Reply:
I'm not sure how locking around ifs works here, and the reader might not either (if I'm already a medium-advanced C++ user). Readability of locking code takes priority here.